HW11/homework_11.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+
+
+class ContextManager:
+    def __enter__(self):
+        print("="*10)
+        return True
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if exc_tb is None and exc_type is None and exc_val is None:
+            print("=" * 10)
+        else:
+            print(exc_type)
+            print(exc_val)
+            print(exc_tb)
+            print("=" * 10)
+            return True
+
+
+with ContextManager():
+    print(1/0)
+
+    def name_time_decorator(func):
+        def inner1(*args, **kwargs):
+            print(f"Function {func.__name__} started at {datetime.now()}")
+            func(*args, **kwargs)
```

Comment:
You misunderstood my comment. The main point here is to return the result of a decorated function to be able to work with this result. To make it works you can simply add here `return` statement. Or, as I mentioned before, you can save a result of the fucn call and return it in the end of inner1 function.
```
def inner1(*args, **kwargs):
    print('...')
    return func(*args, **kwargs)
```

or 
```
def inner1(*args, **kwargs):
    result = func(*args, **kwargs)
    print('...')
    return result